tormenta-avro/src/main/scala/com/twitter/tormenta/scheme/avro/AvroScheme.scala

```diff
@@ -19,15 +19,20 @@ package com.twitter.tormenta.scheme.avro
 import com.twitter.bijection.Injection
 import scala.util.{ Failure, Success }
 import com.twitter.tormenta.scheme.Scheme
+import java.nio.ByteBuffer
 
 /**
  * @author Mansur Ashraf
  * @since 9/14/13
  */
 trait AvroScheme[T] extends Scheme[T] {
+  def decodeRecord(buffer: ByteBuffer)(implicit inj: Injection[T, Array[Byte]]): TraversableOnce[T] = {
```

Comment:
I think it is polite to duplicate this:
https://github.com/twitter/bijection/blob/develop/bijection-core/src/main/scala/com/twitter/bijection/BinaryBijections.scala#L67

But maybe we can be sure that it does not matter that we mutate the caller? Can we comment to that effect?